graphql-kotlin-federation/src/main/protobuf/reports.proto

```diff
@@ -0,0 +1,493 @@
+// This file is copied over from the `apollo-engine-reporting` package, which, in turn, derives its information from
+// Apollo's cloud infrastructure. It is expected that as the protobuf definition changes within the underlying system
+// we are reporting to, these libraries will need to be updated in sync.
+// https://github.com/apollographql/apollo-server/blob/master/packages/apollo-engine-reporting-protobuf/src/reports.proto
```

Comment:
Is there someway we can pull this file from NPM so we don't have to manage it here? https://www.npmjs.com/package/apollo-engine-reporting-protobuf

Reply:
Hmmm! Interesting. I can take a look, good thought 👍